uaclient/cli.py

```diff
@@ -477,6 +562,127 @@ def get_valid_entitlement_names(names: "List[str]"):
     return entitlements_found, entitlements_not_found
 
 
+def action_config(args, cfg, **kwargs):
+    """Perform the config action.
+
+    @return: 0 on success, 1 otherwise
+    """
+    parser = get_parser()
+    subparser = parser._get_positional_actions()[0].choices["config"]
+    valid_choices = subparser._get_positional_actions()[0].choices.keys()
+    if args.command not in valid_choices:
+        parser._get_positional_actions()[0].choices["config"].print_help()
+        raise exceptions.UserFacingError(
+            "\n<command> must be one of: {}".format(", ".join(valid_choices))
+        )
+
+
+def action_config_show(args, cfg, **kwargs):
+    """Perform the 'config show' action.
+
+    @return: 0 on success, 1 otherwise
+    """
+    col_width = str(max([len(x) for x in config.UA_CONFIGURABLE_KEYS]) + 1)
+    row_tmpl = "{key: <" + col_width + "}: {value}"
+    for key in config.UA_CONFIGURABLE_KEYS:
+        print(row_tmpl.format(key=key, value=getattr(cfg, key)))
```

Comment:
This is cool - I haven't used python's padding formatting feature before.

On the other hand, could we get away with just printing yaml? (is that easier or harder than this?)

Reply:
Yep, that makes sense. :+1: